internal/nodes/validator.go

```diff
@@ -361,14 +365,31 @@ func (v *Validator) findTargetPort(serviceName, ns string, servicePort int32, ke
 	}
```

Comment:
Is line 358's check of `NodePort` still needed?

Reply:
No harm in keep/removing the condition check as the `TargetPort` is not getting used for `NodePort`